tutorials/getting_started/another_model_example.md

```diff
@@ -0,0 +1,611 @@
+some larger model.
+
+Next, let's look at the types of the inputs.  The label is simple: it's just a tensor of shape
+`(batch_size, 1)`, with one label id for each instance in the batch.  The other two are a little
+more complicated.  Remember that the title and abstract were `TextFields`; those get converted into
+a dictionary of pytorch tensors.  It's a dictionary instead of a single tensor to be flexible about
+how exactly words are represented in your model.  One element of this dictionary might have a
+tensor of word ids, one might have a tensor of character ids for each word, and one might have a
+tensor of part of speech tag ids.  But your model doesn't have to care about what exactly is in
+that dictionary, because it just passes the dictionary on to the `TextFieldEmbedder` that we took
+as a constructor parameter.  You need to be sure that the `TextFieldEmbedder` is expecting the same
+thing that your `DatasetReader` is producing, but that happens in the configuration file, and we'll
+talk about it later.
+
+Ok, so the first thing that the model does is embed the title and the abstract, then encode them as
+single vectors.  In order to encode the title and abstract, we need to get masks representing which
+elements of the token sequences are merely there for padding.
+
+Once we have a (batched) single vector for the title and the abstract, we concatenate the two
+vectors and pass them through a feed-forward network to get class logits.  We pass the logits
+through a softmax to get prediction probabilities.  Lastly, if we were given a label, we can
+compute a loss and evaluate our metrics.
+
+There are only two other small pieces of code in the `Model` class.  The first is the
+`get_metrics` method:
+
+```python
+    def get_metrics(self, reset: bool = False) -> Dict[str, float]:
+        return {metric_name: metric.get_metric(reset) for metric_name, metric in self.metrics.items()}
+```
+
+This method is how the model tells the training code which metrics it's computing.  The second
+pieces of code is the `decode` method:
+
+```python
+    def decode(self, output_dict: Dict[str, torch.Tensor]) -> Dict[str, torch.Tensor]:
+        predictions = output_dict['class_probabilities'].cpu().data.numpy()
+        argmax_indices = numpy.argmax(predictions, axis=-1)
+        labels = [self.vocab.get_token_from_index(x, namespace="labels")
+                  for x in argmax_indices]
+        output_dict['label'] = labels
+        return output_dict
+```
+
+The name here is a little misleading; originally, we were using this for doing things like viterbi
+decoding or beam search for more complex models, but we now do that in `forward`, typically.
+`decode` is currently just used for making a model's output human-readable (e.g., for the demo).
+We use the vocabulary and translate ids that we see to strings.
+
+Alright, we've got ourselves a model.  We can run the tests again using `pytest`, and confirm that
```

Comment:
you've used this "Alright," a couple of times

Reply:
Yeah, it was very late as I was writing this, and I didn't ever give it another read-through.  I'll go over it again to fix some of these wording issues.